docs/pages/ESSupport.md

```diff
@@ -0,0 +1,368 @@
+
+## ECMAScript Version Support
+
+The UI5 Tooling offers general support for the latest ECMAScript features. While a `ui5 build` is executed, the UI5 Tooling analyses the code in a few places. Depending on your project you have to consider some restrictions regarding the used ECMAScript syntax.
```

Comment:
> The UI5 Tooling offers general support for the latest ECMAScript features

Currently in v3 we are supporting ES2021 (see https://github.com/SAP/ui5-builder/blob/31655b603e0de2c5f2e44ae5a9cbdd9019b7765c/lib/lbt/utils/parseUtils.js#L10). AFAICS ES2022 is final, so as a read I would expect that as the currently latest version of features.

> Depending on your project

Depending on what specific thing about a project? I find that reference a bit confusing.
EDIT: Now after reading the guide I see that it's specific to the project type.

Reply:
we only analyse js files declared as "script". "module" are not tackled by the UI5 Tooling

Reply:
Put ES2022 instead of latest

Reply:
done